pandas/core/series.py

```diff
@@ -274,8 +274,8 @@ def __init__(
                 try:
                     if len(index) != len(data):
                         raise ValueError(
-                            "Length of passed values is {val}, "
-                            "index implies {ind}".format(val=len(data), ind=len(index))
+                            f"Length of passed values is {len(data)}, "
+                            "index implies {len(index)}."
```

Comment:
```suggestion
                            f"index implies {len(index)}."
```
needs an `f` here too.